test/python/WMCore_t/WorkQueue_t/WorkQueue_t.py

```diff
@@ -602,44 +611,43 @@ def testQueueChaining(self):
                                        {})
         self.assertEqual(0, len(self.localQueue))
         self.assertEqual(2, len(work))
-
+ 
```

Comment:
What's going on here? Your editor seems to be inserting a blank space in a lot of empty lines. Can you update the patch stripping all trailing space?